{{cookiecutter.project_name}}/.pre-commit-config.yaml

```diff
@@ -54,3 +54,10 @@ repos:
     rev: v{{ cookiecutter._vulture_version }}
     hooks:
       - id: vulture
+{%- if cookiecutter.__project_type=="agent_plugin" %}
+  - repo: https://github.com/guardicode/plugin-version-commit-hook
+    rev: v1.0.0
```

Comment:
The new hook is at 1.0 already?

Reply:
This needs to be removed. Our plugins should not be packages and not have a version. Commit is coming.